src/display.jl

```diff
@@ -400,8 +407,8 @@ end
 
 
 """
-    record(func, scene, path; framerate = 24)
-    record(func, scene, path, iter; framerate = 24)
+    record(func, scene, path; framerate = 24, kwargs...)
```

Comment:
What are these kwargs for? They should be written out or documented!

Reply:
Will do, I left that in from when I had 5 bitrate-related kws :D